bindings/python/test/environment/__init__.py

```diff
@@ -6,3 +6,9 @@
 # @license        Apache License 2.0
 
 ################################################################################################################################################################
+
+import pytest
+
+import ostk.physics as physics
+
+################################################################################################################################################################
```

Comment:
Is this necessary?

Reply:
I don't think it is. @apaletta3 Did you apply the previous suggestions to remove this? 🤔 (I might have missed a file or 2 when adding those suggestions)

Reply:
fixed